src/Core/ServiceWrapper/CLI/TestOption.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CommandLine;
+using System.Threading;
+using WMI;
+
+namespace winsw.CLI
+{
+    [Verb("test", HelpText = "check if the service can be started and then stopped")]
+    public class TestOption : CliOption
+    {
+        public override void Run(ServiceDescriptor descriptor, Win32Services svcs, Win32Service? svc)
+        {
+            if (!Program.elevated)
+            {
+                Elevate();
+                return;
+            }
+
+            var arguments = Parser.Default.FormatCommandLine(Program.cliOption).Split(' ');
```

Comment:
This doesn't work if the path contains spaces.

Reply:
As I know internally those arguments are not used. I doubt that which kind of argument array will be fit to this. At the moment this is the argument structure.
`winsw install --configFile <filepath> --elevated true -s true`

How should the items be separated here. I found in the commandline library. There is no way to unprase the object back into arguments array. So I think I have to manually convert it?

Reply:
> internally those arguments are not used.

OK. Then passing an empty array would be preferred.

Reply:
Make sense! I will do so.